### ISSUE TITLE
pdf created by bot cant get past captcha

### ISSUE THREAD

**Issue by Marsinator** [neopostmodern/wohnungsbot#201]
this is what I get when I hit the pdf button created by the bot for the ad that it applied to

![image](https://github.com/neopostmodern/wohnungsbot/assets/7913458/e28d9781-7891-4b2e-9d2a-389d6286da8e)

**Comment by neopostmodern** [comment]
Yeah, this is a complicated edge-case that won't get fixed on it's own I think. The only option is to hope for #87...
